dev/PushNotifications/PushNotificationsLongRunningTask.ProxyStub/NotificationsLongRunningProcess.idl

```diff
@@ -6,12 +6,25 @@ import "ocidl.idl";
 
 #include "../PushNotifications-Constants.h"
 
+[object]
+[uuid(25604D55-9B17-426F-9D67-2B11B3A65598)]
+[pointer_default(unique)]
+interface IWpnForegroundSink : IUnknown
+{
+    HRESULT InvokeAll([in] ULONG length, [in, string] byte data[]);
```

Comment:
'string' means zero-terminated for length. I think you meant `[in, size_is(length)] byte*` ?